src/components/views/messages/ReactionsRowButton.tsx

```diff
@@ -105,22 +107,50 @@ export default class ReactionsRowButton extends React.PureComponent<IProps, ISta
         }
 
         const room = this.context.getRoom(mxEvent.getRoomId());
+        const customReactionImagesEnabled = SettingsStore.getValue("feature_render_reaction_images");
         let label: string | undefined;
+        let customReactionName: string | undefined;
         if (room) {
             const senders: string[] = [];
             for (const reactionEvent of reactionEvents) {
                 const member = room.getMember(reactionEvent.getSender()!);
                 senders.push(member?.name || reactionEvent.getSender()!);
+                if (customReactionImagesEnabled && reactionEvent.event.content?.["com.beeper.reaction.shortcode"]) {
```

Comment:
Yeah, that would be nice

Reply:
It is an unstable value. It will be `shortcode` when it's stable: https://github.com/matrix-org/matrix-spec-proposals/pull/4027/files#diff-c075783679f0bce493f0a817a6f71200aa434dd8ac13fef5265089a8dc433506R126-R127

Reply:
Oh, I see. Done.